lib/cbra/change.rb

```diff
@@ -9,14 +9,34 @@ def initialize(path, option, branch)
       @path = path
       @option = option
       @branch = branch
+      @tree = ComponentTree.new(path).to_h
     end
 
     def run!
       return unless valid?
+
+      changes_since_last_commit
+      directly_affected_components
     end
 
   private
 
+    def changes_since_last_commit
+      puts "<<<Changes since last commit on #{@branch}>>>"
+      puts blank_line
+      root_dir = `cd "#{@path}" && git rev-parse --show-toplevel`.chomp
+      @changes = `cd "#{root_dir}" && git diff --name-only #{@branch}`.split("\n").map { |f| File.join(root_dir, f) }
```

Comment:
I need it here though (as things stand now) - https://github.com/powerhome/cbra/pull/4/files#diff-2bb9734b5576a0f933e1488dadb4b218R62